src/sentry/templates/sentry/emails/mfa-too-many-attempts.html

```diff
@@ -0,0 +1,16 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Suspicious Activity Detected</h3>
+    <p>This is a notice that your Sentry account ({{ email }}) has failed multiple login attempts at the 2FA step from the following location:</p>
+    <p><small><pre>
```

Comment:
Why the `small` tag? Our other email templates that include a preformatted data block like this don't use `small`.

Reply:
Took it from https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/recover_account.html#L13
but I agree, this `small` tag does not even have effect.